dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -332,10 +332,13 @@ public void testComponent_success() throws IOException, ParsingException {
   public void testComponent_failure() throws IOException, ParsingException {
     Document document = parseOutputFile(
         "com.google.api.grpc_grpc-google-common-protos_1.14.0.html");
+
+    // com.google.api.grpc:grpc-google-common-protos:1.14.0 selects the highest version of all dependencies.
     Nodes greens = document.query("//h3[@style='color: green']");
-    Assert.assertEquals(0, greens.size());
+    Assert.assertEquals(1, greens.size());
```

Comment:
Before this fix [the artifact report for grpc-google-common-protos](https://storage.googleapis.com/cloud-opensource-java-dashboard/com.google.cloud/libraries-bom/1.0.0/com.google.api.grpc_grpc-google-common-protos_1.14.0.html
) was showing Upper Bound Fixes for jsr305 version 3.0.1. But that version should not appear in the tree because grpc-core declares exclusions.

Tree: https://screenshot.googleplex.com/g8cQ3BXq36O.png

Reply:
The comment above was correct but `grpc-google-common-protos:1.14.0`'s dependency tree still has old protobuf-java dependency.